src/databricks/labs/ucx/queries/progress/02_1_migration_status_by_owner_bar_graph.sql

```diff
@@ -0,0 +1,22 @@
+/*
+--title 'Tables and views not migrated'
+--overrides '{"spec": {
+    "version": 3,
+    "widgetType": "bar",
+    "encodings": {
+        "x":{"fieldName": "owner", "scale": {"type": "categorical"}, "displayName": "owner"},
+        "y":{"fieldName": "count", "scale": {"type": "quantitative"}, "displayName": "count"}
+    }
+}}'
+*/
+WITH migration_statuses AS (
+    SELECT *
+    FROM inventory.historical
+    WHERE object_type = "migration_status"
```

Comment:
This needs to:
a) Join against the `workflow_runs` table;
b) Only select rows that belong to the latest run for each workspace.